Makefile

```diff
@@ -276,6 +271,15 @@ jstest: build-addons build-js-native-api-tests build-node-api-tests ## Runs addo
 		$(CI_JS_SUITES) \
 		$(CI_NATIVE_SUITES)
 
+.PHONY: jscov
+jscov:
+	$(RM) -r out/$(BUILDTYPE)/.coverage
+	$(MAKE) coverage-build-js
+	-NODE_V8_COVERAGE=out/$(BUILDTYPE)/.coverage \
+	  CI_SKIP_TESTS=core_line_numbers.js,testFinalizer.js,test_function/test.js \
```

Comment:
Could the list of tests be refactored to a new var (`TEST_TO_SKIP_FOR_COVERAGE`)? and D.R.Y. L307?

Reply:
I wonder if we could use `TEST_TO_SKIP_FOR_COVERAGE` to isolate the tests that fail under coverage (@mhdawson?) and stop using `-` to suppress erros.